command/operator_client_state.go

```diff
@@ -0,0 +1,115 @@
+package command
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-hclog"
+	trstate "github.com/hashicorp/nomad/client/allocrunner/taskrunner/state"
+	"github.com/hashicorp/nomad/client/state"
+	"github.com/hashicorp/nomad/nomad/structs"
```

Comment:
Hi @tgross , could you help me review this MR, especially the part about failed `lint-go` test ([link](https://app.circleci.com/pipelines/github/hashicorp/nomad/35807/workflows/d6ea1e69-7a9b-4d12-a85f-a2f85b4703ee/jobs/394469)) ?. I think the situation is not as simple as changing `structs` to `api` like below ( ref #13938 )
```
type clientStateAlloc struct {
	Alloc        *api.Allocation
	DeployStatus *api.AllocDeploymentStatus
	Tasks        map[string]*taskState
}

type taskState struct {
	LocalState  *trstate.LocalState
	RemoteState *api.TaskState
	DriverState interface{}
}
```
Its because we are using `GetDeploymentStatus()` which returns a `*structs.AllocDeploymentStatus` too. Could you share with me what you have in mind on this too ? Thanks alot !